### ISSUE TITLE
nwb v0.21.5 [[object Set]] error

### ISSUE THREAD

**Issue by reviewher** [insin/nwb#452]
Just upgraded to latest version, the project type list is incorrectly displayed as `[[object Set]]`:

```
$ nwb --version
v0.21.5
$ node --version
v9.11.1
$ npm --version
5.6.0
$ nwb new
usage: nwb new [[object Set]] <name>
$ nwb new wtf
Project type must be one of: [object Set]
```

**Comment by haani104** [comment]
Hi, @insin Any plan for the fix?

**Comment by insin** [comment]
Looks like this was already fixed in v0.22.0

```sh
λ nwb v
v0.22.0

λ nwb new
usage: nwb new [inferno-app|preact-app|react-app|react-component|web-app|web-module] <name>
```
